csv-bulk-signature/src/main.ts

```diff
@@ -102,8 +113,13 @@ function generateSignatures(
       console.log("Number of signatures processed", processedRows.length);
       console.log("Number of signatures skipped", skippedRows.length);
       createStatusReport(); // refactoring the results specifics to be shown in a text file, instead
+
+      // wait three seconds before zipping up the files
+      console.log("Zipping up files...");
+      setTimeout(() => {
+        zipUpFile();
+      }, 1500);
     });
-  zipUpFile();
```

Comment:
don't zip until we have all signatures created --- hack by waiting a couple seconds